packages/odyssey/src/scss/components/_tab.scss

```diff
@@ -24,7 +24,7 @@
   background: none;
   color: $text-body;
   font-family: inherit;
-  font-size: $size-title-5;
+  font-size: $size-title-6;
```

Comment:
Is this necessary? Base font-size is 16px. 

I think we can scrap this.

Reply:
I think this is a question for @joejezowski-okta - if our Title 6 doesn't always match Body, do we want Tabs to use the smallest possible title or the body size?